src/OSPSuite.Presentation/Presenters/AbstractExplorerPresenter.cs

```diff
@@ -324,6 +332,12 @@ public virtual void AddClassifiableNodeToView(ITreeNode classifiableNode, ITreeN
             _view.AddNode(classifiableNode);
       }
 
+      public virtual IEnumerable<IMenuBarItem> AllCustomMenuItemsFor(ClassificationNode classificationNode)
```

Comment:
@Yuri05 @rwmcintosh Another place where we need some customizable way of adding menus. By default nothing added but pksim (or mobi) can add generic menus based on the classification